packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -323,7 +323,7 @@ class StripeHelper {
    */
   async refreshCachedCustomer(uid, email) {
     try {
-      return await this.customer(uid, email, true);
```

Comment:
Removing the `await` here means that the try/catch never catches an exception. The unresolved failure bubbles to the caller. If that's intentional, we should remove the try/catch

Reply:
Woops! Fixed.